### ISSUE TITLE
clarity keyboard focus ring not working on articles page filter

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5472]
1.9 rc1

<img width="428" height="428" alt="Image" src="https://github.com/user-attachments/assets/52d3def7-799d-4661-85a6-cc783b451a61" />

The two colour keyboard focus indicator we use to show it will work on any background is missing from the filter inputs in Clarity.

**Comment by ajrbyers** [comment]
Closed by https://github.com/openlibhums/janeway/pull/5521
